packages/validator/src/plugins/validation/2and3/semantic-validators/schema-ibm.js

```diff
@@ -194,11 +194,15 @@ function typeFormatErrors(obj, path, isOAS3, messages, checkStatus) {
   const validIntegerFormats = ['int32', 'int64'];
   const validNumberFormats = ['float', 'double'];
   const validStringFormats = [
-    'byte',
     'binary',
+    'byte',
+    'crn',
     'date',
     'date-time',
-    'password'
+    'email',
+    'identifier',
+    'password',
+    'uuid'
```

Comment:
We already support "uuid" in the generator as an official format value for string schemas.